indykite/common_schema.go

```diff
@@ -48,15 +50,41 @@ const (
 
 func regionSchema() *schema.Schema {
 	return &schema.Schema{
-		Type:        schema.TypeString,
-		Required:    true,
-		Description: `Region where the application space is located.`,
+		Type:     schema.TypeString,
+		Required: true,
+		Description: `Region where the application space is located.
+		Valid values are: europe-west1, us-east1.`,
 		ValidateFunc: validation.StringInSlice([]string{
 			"europe-west1", "us-east1",
 		}, false),
 	}
 }
 
+func ikgSizeSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeString,
+		Optional: true,
+		Description: `IKG size that will be allocated, which correspond also to amount of CPU nodes (default 2GB).
+		Valid values are: 2GB, 4GB, 8GB, 16GB, 32GB, 64GB, 128GB, 192GB, 256GB, 384GB, 512GB.`,
+		ValidateFunc: validation.StringInSlice([]string{
+			"2GB", "4GB", "8GB", "16GB", "32GB", "64GB", "128GB", "192GB", "256GB", "384GB", "512GB",
+		}, false),
+	}
+}
+
+func replicaRegionSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeString,
+		Optional: true,
+		Description: `Replica region specify where the second IKG is created.
+		Replica must be different region than the master, but also in the same geographical continent.
```

Comment:
"Replica must be **a** different region..."

"[..] but also ~in~ **on** the same geographical continent.